sda-commons-server-kafka/src/test/java/com/salesforce/kafka/test/SharedKafkaTestResourceScram.java

```diff
@@ -0,0 +1,100 @@
+package com.salesforce.kafka.test;
+
+import com.salesforce.kafka.test.junit4.SharedKafkaTestResource;
+import com.salesforce.kafka.test.junit4.SharedZookeeperTestResource;
+import com.salesforce.kafka.test.listeners.BrokerListener;
+import com.salesforce.kafka.test.listeners.SaslScramListener;
+import java.lang.reflect.Field;
+import java.lang.reflect.Modifier;
+import kafka.admin.ConfigCommand;
+import org.junit.runner.Description;
+import org.junit.runners.model.Statement;
+
+/**
+ * Creates and starts up a {@link SharedKafkaTestResource} that uses a custom {@link
+ * SharedZookeeperTestResource} with installed SCRAM-SHA-* credentials.
+ *
+ * <p>See {@link SharedKafkaTestResource} for usage.
+ */
```

Comment:
Have you tried contributing the class to the original test library?

Reply:
Not yet. It would be nice if we integrate this into the original library since we could then get rid of the quite hacky code in SharedKafkaTestResourceScram.java. But we should do this independent from this PR.